common/config/rush/common-versions.json

```diff
@@ -49,12 +49,6 @@
     // "typescript": [
     //   "~2.4.0"
     // ]
-    // service-bus is not ready to update to @azure/core-amqp@1.0.0-preview.7
-    "@azure/core-amqp": ["1.0.0-preview.3"],
-    "@azure/ms-rest-js": ["^2.0.0"],
```

Comment:
Did you intentionally delete @azure/ms-rest-js reference too?

Reply:
A duplicate entry exists on line 44
Likely a miss while updating the base branch cc: @ramya-rao-a

Reply:
@ramya-rao-a any comments since you had updated the base branch manually?
From response above, it's unclear if something still needs fixing..

P.s this is why it would have been better if you sent out a PR for the changes instead, than manually update - so misses as these can be caught early on

Reply:
Nope, no fixing needed here. Resolving comment.

Reply:
Nope, no fixing needed here. Resolving comment.